### PR TITLE
refactor(client/commands): qb-vehiclekeys:client:GiveKeys to separate file

### PR DESCRIPTION
**Pull request by ArturMichalak**
## Description

separating command-related functions from the rest of the code will improve code readability

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [X] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: client/commands.lua
Comment (Manason) [inline client/commands.lua:1]: Maybe rename this file to give-keys.lua? This happens to be invoked by a command, but it isn't actually a command and could also be invoked programmatically, or the command removed without changing this code.

**Review by Manason — APPROVED** [review]

File: client/commands.lua
Comment (ArturMichalak) [reply client/commands.lua:1]: The code is strictly related to the command. I don't see any problems with renaming the file, but I'm not sure if it is a good option, when most of the functions are in the functions.lua file. I want to isolate the functions from the rest of the resource.

Comment (Manason) [reply client/commands.lua:1]: It's fine if you want to keep it. I just don't see how it's related to a command. Aren't we going to have a GiveKey export which invokes the same function/event?

Comment (ArturMichalak) [reply client/commands.lua:1]: I don't expect such an option. In case any of these functions are needed somewhere else it will be moved to functions.lua

**Pull request merged**